src/base/header.rs

```diff
@@ -286,7 +286,7 @@ impl Header {
     /// [`Rcode`]: ../../iana/rcode/enum.Rcode.html
     #[must_use]
     pub fn rcode(self) -> Rcode {
-        Rcode::from_int(self.inner[3] & 0x0F)
+        Rcode::saturating_from_int(self.inner[3])
```

Comment:
Why is this called saturating _from_int. It seems to just mask the lower 4 bits. What is the saturating part?

Reply:
The idea was to have a name reminiscent of something like `saturating_sub` which accepts the input and sanitizes it into a valid value which is what this does as well.

Reply:
Maybe call it masked_from_int?

Reply:
That sounds like a good idea. Will do!